### PR TITLE
Update proxy to handle port redirects

### PR DESCRIPTION
**Pull request by marcosnils**
@xetorthio :bell:

### CODE REVIEW

**Comment by xetorthio** [comment]
What is this for?

**Comment by marcosnils** [comment]
@xetorthio ???

**Comment by marcosnils** [comment]
@xetorthio :bell:

File: handlers/reverseproxy.go [hunk 1]
```diff
@@ -30,9 +31,15 @@ func NewMultipleHostReverseProxy() *httputil.ReverseProxy {
 		v := mux.Vars(req)
 		node := v["node"]
 		port := v["port"]
-		if port == "" {
+		hostPort := strings.Split(req.Host, ":")
```
Comment (marcosnils) [inline handlers/reverseproxy.go]: I didn't use mux vars because host port can't be parsed there.

**Pull request merged**